packages/turbo-benchmark/src/index.ts

```diff
@@ -1,8 +1,17 @@
 import cp from "node:child_process";
-import fs from "node:fs";
 import path from "node:path";
-import fse from "fs-extra";
-import ndjson from "ndjson";
+import {
+  copySync,
+  mkdirSync,
+  rmSync,
+  writeFileSync,
+  readFileSync,
+  existsSync,
+  closeSync,
+  openSync,
+  writeSync,
+} from "fs-extra";
```

Comment:
Can we keep the `node:fs` imports for the ones that were previously using them? I'd prefer not to use 3rd party libraries (even if they're just proxying back to the standard lib)

Reply:
@mehulkar we could, sure - but it's more confusing to use two different FS libs. We are already installing it, why not use it for everything?

Reply:
I would prefer to be explicit about when the built in std library doesn't cover the functionality we need. It would be somewhat self-documenting why we chose to use the 3rd party library. If we universally use the 3rd party library, it will be difficult to understand later whether it's _actually_ necessary or not.

Reply:
Fair enough, for something internal I don't think it's necessary, dev prod > bundle size. But can remove if you prefer.

Reply:
@mehulkar updated